maxwellgithinji/StackOverflowRevamped-API
# #166484644 CRUD Answers

#### What does this PR do?
- ensure a user can comment on an answer
- ensure o user is only able to edit or delete an answer they authored
#### Description of Task to be completed?
- add CRUD functionality to answers
#### How should this be manually tested?
- checkout `ft-crud-answers-166484644` 
- refer to the readme for the appropriate endpoints
- signup to the application
- post a question
- post an answer to the question
- if the author of the answer, you should be able to edit and delete answers

#### Any background context you want to provide?
N/A

#### What are the relevant PT Stories?
[#166484644](https://www.pivotaltracker.com/story/show/166484644)
#### Screenshots (if appropriate)
- Add screenshot
#### Questions:
